website/docs/d/file.html.md

```diff
@@ -42,17 +40,17 @@ data "ignition_file" "hello" {
 
 The following arguments are supported:
 
-* `filesystem` - (Required) The internal identifier of the filesystem. This matches the last filesystem with the given identifier. This should be a valid name from a _ignition\_filesystem_ resource.
-
 * `path` - (Required) The absolute path to the file.
 
+* `overwrite` - (Optional) Whether to delete preexisting nodes at the path. Defaults to true.
```

Comment:
Please double check. I believe in Ignition v2 / spec v3.x, this defaults to `false`. (used to be `true` in v0.x / spec 2.x)

Reply:
that seems correct according to https://github.com/coreos/ignition/blob/master/docs/configuration-v3_2.md
I've updated it